src/components/Checkbox/Checkbox.tsx

```diff
@@ -0,0 +1,63 @@
+import React from "react";
+
+interface Props {
+  id: string;
+  isTile?: boolean;
+  label: string;
+  name: string;
+  tileDescription?: string;
+  value?: string;
+}
+
+/**
+ * Checkbox Component
+ * @param {string}  id                A unique identifier for the input.
+ * @param {string}  label             Label text that appears to the right of the Checkbox.
+ * @param {string}  name              Name of the input element.
+ * @param {boolean} [isTile]          Use the tile variation of the Checkbox.
+ * @param {string}  [tileDescription] Text that can be used to describe the label in more detail. Activates the tile variation automatically.
+ * @param {string}  [value]             Value of the input element.
+ */
+export const Checkbox: React.FC<Props> = ({
+  id,
+  isTile,
```

Comment:
I'm thinking for booleans lets default them instead of leaving them undefined. This is purely for consistency and it looks slightly nice to have the default column on the storybook as false instead of "-"

![Screen Shot 2022-07-27 at 2 03 40 PM](https://user-images.githubusercontent.com/55826377/181371925-11337328-e56b-4897-a387-bf020383af09.png)


isTitle= false,

Reply:
Bump on this one